utils/inotify-tools/Makefile

```diff
@@ -25,6 +25,7 @@ endif
 
 ## Avoid linking with libstdcpp
 TARGET_CXXFLAGS+= -nodefaultlibs -lc -fno-exceptions
+TARGET_LDFLAGS += $(if $(CONFIG_USE_MUSL),-lssp_nonshared)
```

Comment:
Cosmetic note only. Could you please remove the <space> between `TARGET_LDFLAGS` and `+=`

Reply:
Good catch, done